Yommy, we should merge this if we want send our latest changes upstream

brackendawson opened the pull request.

brackendawson committed:
Lots of changes/fixes to tron by Paul & I

Yommy committed:
restored Copyright

Yommy commented:
i still have some changes being worked at https://gist.github.com/Yommy/cbd4a239d9c104a9278d

Yommy committed:
Update tron.py

added an alternate score system, score = deaths - kills
better avoidance of 180 suicide
game length back to 5 minutes

Yommy committed:
Update tron.py

Added connection details before game, with a countdown timer to game start
changed the score system to kills-deaths
fixed some bugs
